_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
```

Comment:
Ah yes, oversight on my part. Why can't we just create an `id` on the `input` that matches the `for` on the `label` (`veteran_mothers_maiden_name`)? That would associate them without needing to include a unique id number. I thought the unique id was in case we reuse the component as is for multiple fields, which I don't think we would do in this case.